fenix/app/src/main/res/values/strings.xml

```diff
@@ -1308,6 +1308,8 @@
     <string name="onboarding_firefox_account_sync_is_on">Sync is on</string>
     <!-- Text for the tracking protection onboarding card header -->
     <string name="onboarding_tracking_protection_header">Privacy protection by default</string>
+    <!-- Text for the tracking protection card description. The first parameter is the name of the application.-->
+    <string name="onboarding_tracking_protection_description_old">%1$s automatically stops companies from secretly following you around the web.</string>
```

Comment:
Using the same string as before - https://github.com/mozilla-mobile/firefox-android/commit/b2845db6f9d56769a697dd062453f203e57ec36b?diff=unified#diff-b24d2928f9f4594421c030b62d3e5e070c9fd5600c871ff9a8c24a518fd02f96R1208